lib/EmbedCodeGenerator.js

```diff
@@ -457,7 +457,10 @@ EmbedCodeGenerator.prototype = {
 			data['embedMethod'] = (params.embedType == 'dynamic') ? 'embed' : 'thumbEmbed';
 			data['kWidgetObject'] = this.getEmbedObject( params );
 		}
-
+        var date = parseInt(params.entryMeta.uploadDate);
+        if (date.toString().length === 10) {
```

Comment:
not clear what this check is? also, why do you move from sting to int the line above just to parse to string back here?